internal/http/services/owncloud/ocs/handlers/apps/sharing/shares/shares.go

```diff
@@ -52,17 +54,18 @@ import (
 
 // Handler implements the shares part of the ownCloud sharing API
 type Handler struct {
-	gatewayAddr         string
-	publicURL           string
-	sharePrefix         string
-	homeNamespace       string
-	userIdentifierCache *ttlcache.Cache
+	gatewayAddr            string
+	publicURL              string
+	sharePrefix            string
+	homeNamespace          string
+	additionalInfoTemplate *template.Template
+	userIdentifierCache    *ttlcache.Cache
 }
 
 // we only cache the minimal set of data instead of the full user metadata
 type userIdentifiers struct {
 	DisplayName string
-	UserName    string
+	Username    string
```

Comment:
Follow [Go's mixedCaps](https://golang.org/doc/effective_go#mixed-caps) style? 😇

Reply:
I changed it so that we can use the same layout for shares and shares. Sharees uses executes the template on the CS3APIs user object, which has the 'Username' parameter (https://github.com/cs3org/reva/blob/master/internal/http/services/owncloud/ocs/handlers/apps/sharing/sharees/sharees.go#L119), so I changed it to make it compatible

Reply:
0Oh thanks for clarifying :)